de-lib/src/main/java/org/iplantc/de/commons/client/widgets/DETabPanel.java

```diff
@@ -36,4 +36,9 @@ public void setTabDebugId(Widget widget, String debugId) {
         XElement item = findItem(getWidgetIndex(widget));
         item.setId(debugId);
     }
+
+    @Override
+    public void close(Widget item) {
+        super.close(item);
+    }
```

Comment:
This method in `TabPanel` is protected.  I think the only other way to get access to this method is to have those little close buttons per tab, which I didn't like.